src/ode_premade_problems.jl

```diff
@@ -321,7 +321,7 @@ pleiades = (t,u,du) -> begin
 end
 tspan= (0.,3.)
```

Comment:
Oh, that is because I just noticed it wasn't even exported in the first place. Going to add it to the list of exported functions in `DiffEqProblemLibrary.jl`